source/js/components/petition/locale-strings.jsx

```diff
@@ -122,7 +188,29 @@ export default {
     "Please check this box if you want to proceed": `Veuillez cocher cette case si vous désirez poursuivre`,
     "Yes, I want to receive email updates about Mozilla's campaigns.": `J’accepte de recevoir des informations par courriel au sujet des campagnes de Mozilla.`,
     "Add my name": `Ajouter mon nom`,
-    "Sign up": `Je m’inscris`
+    "Sign up": `Je m’inscris`,
+    "If you haven’t previously confirmed your opt-in to a Mozilla-related email subscription you may have to do so now.": (
```

Comment:
@Pomax do you know where we can get the actual localized strings from? Do we have a system in place that auto updates strings here?

Reply:
No, these were all temporary work that we had localisers do for us because it was far less work to do the set of at-the-time fixed strings than it was to figure out a true localisation strategy. The work to do this properly is currently being investigated by @TheoChevalier